src/test/java/io/debezium/connector/yugabytedb/YugabyteDBSnapshotTest.java

```diff
@@ -312,7 +347,12 @@ public void snapshotColocatedNonColocatedThenStream() throws Exception {
         TestHelper.executeInDatabase("INSERT INTO test_3 VALUES (generate_series(3000, 6000));", DEFAULT_COLOCATED_DB_NAME);
 
         List<SourceRecord> records = new ArrayList<>();
-        waitAndFailIfCannotConsume(records, recordCountForTest1 + recordCountForTest2 + recordCountForTest3 + recordCountInNonColocated + 1001 + 3001);
+        if(initial_only){
```

Comment:
We have followed the coding standard where there's a space after `if` and `else` starts on the same line as the ending of if-block, like:
```
if (initial_only) {
  // If block content
} else {
  // else block content
}
```